docs/en/pact-functions.md

```diff
@@ -1658,6 +1676,18 @@ Query gas state, or set it to GAS.
 Set environment gas limit to LIMIT.
 
 
+### env-gaslog {#env-gaslog}
+
+ *&rarr;*&nbsp;`string`
+
+
+Enable and obtain gas logging. Bracket around the code whose gas logs you want to inspect.
+```lisp
+pact> (env-gasmodel "table") (env-gaslimit 10) (env-gaslog) (map (+ 1) [1 2 3]) (env-gaslog)
+["TOTAL: 7" "map:GUnreduced: 4" "+:GUnreduced: 1" "+:GUnreduced: 1" "+:GUnreduced: 1"]
+```
+
+
 ### env-gasmodel {#env-gasmodel}
 
 *model*&nbsp;`string` *&rarr;*&nbsp;`string`
```

Comment:
Forgot to mention in previous review, but we should give an example that uses `(env-gasmodel "table")`, otherwise users might not know that that's the name of the gas model used in Chainweb.